docs/code-quality/ca2248.md

```diff
@@ -0,0 +1,56 @@
+---
+title: 'CA2248: Provide correct 'enum' argument to 'Enum.HasFlag''
```

Comment:
```suggestion
title: "CA2248: Provide correct 'enum' argument to 'Enum.HasFlag''"
```

Reply:
Change to fix a rendering issue

Reply:
We want to avoid inline styles in metadata and headings. For inline code style, we are using the backtick `